### PR TITLE
community/coturn: upgrade to 4.5.1.1

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

**Comment by ad-m** [comment]
CC: @clandmeter

**Comment by TBK** [comment]
All builddir can be removed.

**Comment by ad-m** [comment]
Fixed

File: community/coturn/APKBUILD [hunk 1]
```diff
@@ -20,7 +20,6 @@ source="coturn-$pkgver.tar.gz::https://github.com/coturn/coturn/archive/$pkgver.
 builddir="$srcdir/coturn-$pkgver"
```
Comment (TBK) [inline community/coturn/APKBUILD]:
```suggestion

```

**Pull request closed without merging**